eng/tools/generator/repo/github.go

```diff
@@ -0,0 +1,55 @@
+package repo
```

Comment:
Is this file useless if we use ps scripts?

Reply:
removed github.go file